lib/features/digital_guide_view/presentation/widgets/accessibility_button.dart

```diff
@@ -0,0 +1,26 @@
+import "package:flutter/material.dart";
+
+import "../../../../config/ui_config.dart";
+
+class AccessibilityButton extends StatelessWidget {
```

Comment:
idk if you did this, but i would make it look similar to either the one in SKS Menu or the one in the Last Navigation Tab. Maybe you could reuse some elements / extarct some common base widget?

Reply:
Do you mean the color aspects? Because I tried to follow mock on Figma

Reply:
Following figma is a good thing to do. Don't wanna hurt your feelings, cause you're making a massive work here, but "tried" is prolly the key word here? 

Details are important, and here I've spotted it with blind eye.

E.g. the one on figma has border radius 8 and yours 10. The figma has "mirage" color of icon and your is black. Figma has "pigeon" color and yours is 70% opacity of black. Also padding-wise this seems to be off a bit? But maybe I'm wrong @24bartixx

Reply:
Omg you are completely right... It was my first time using Figma and I didn't realize you can examine parameters of the widgets... I have much more stylistic work to do then than simply this button

Reply:
And don't be afraid to hurt my feelings, work must be done correctly, right?